src/bp/ui-studio/src/web/views/FlowBuilder/common/actionPopover.tsx

```diff
@@ -0,0 +1,59 @@
+import classnames from 'classnames'
+import { parseActionInstruction } from 'common/action'
+import React, { FC, useState } from 'react'
+import { Overlay, Popover } from 'react-bootstrap'
+import ReactDOM from 'react-dom'
+
+import style from './style.scss'
+
+interface Props {
+  text: string
+  className: string
+}
+
+export const ActionPopover: FC<Props> = props => {
+  const [show, setShow] = useState<boolean>(false)
+  const [target, setTarget] = useState<HTMLElement>(null)
+
+  const actionInstruction = parseActionInstruction(props.text.trim())
+  const actionName = `${actionInstruction.actionName} (args)`
+
+  let callPreview: string
+  if (actionInstruction.argsStr) {
+    try {
+      const parameters = JSON.parse(actionInstruction.argsStr)
+      callPreview = JSON.stringify(parameters, null, 2)
+    } catch (err) {
+      console.error(err)
```

Comment:
Simply printing the error will make it hard to track the origin of the error.
You should add some indicator that makes the origin easy to find in the codebase, e.g. `console.error(`[ActionPopover] Error parsing instructions:`, error)`